src/@orb.yml

```diff
@@ -113,24 +113,21 @@ jobs:
     machine:
       image: ubuntu-2204:current
     steps:
-      - when:
-          condition: <<parameters.attach_workspace>>
-          steps:
-            - attach_workspace:
-                at: <<parameters.workspace_root>>
-      - aws-cli/setup:
-          profile_name: global_res
       - aws-ecr/ecr_login:
           profile_name: global_res
```

Comment:
Not sure if that is even needed. IIUC the `build_and_push_image` supports `profile_name`, but I still have to figure out if the outcome will be the same